front/lib/plans/plan_codes.ts

```diff
@@ -8,6 +8,17 @@ export const TRIAL_PLAN_CODE = "TRIAL_PLAN_CODE";
 
 // Current pro plans:
 export const PRO_PLAN_SEAT_29_CODE = "PRO_PLAN_SEAT_29";
+export const PRO_PLAN_OUTLEAP_CODE = "PRO_PLAN_OUTLEAP";
+export const PRO_PLAN_LARGE_FILES_CODE = "PRO_PLAN_LARGE_FILES";
+
+export const PRO_PLANS_CODES = [
+  PRO_PLAN_SEAT_29_CODE,
+  PRO_PLAN_OUTLEAP_CODE,
```

Comment:
What is this one?

Reply:
No idea but is on prod! I found only one papertrail about it that was not about its creation.

Reply:
huh

Reply:
https://dust4ai.slack.com/archives/C05UDJU9JKW/p1700959630986929 linked to this I believe